### DISCUSSION TITLE
Show response schemas next to the response examples

### DISCUSSION THREAD

**Discussion by myftija** [scalar/scalar#329] [Ideas]
Hi 👋

It would be useful to show the schema of the response object instead of only an example response. Maybe somewhat similar to how Github approaches this: [link](https://docs.github.com/en/free-pro-team@latest/rest/repos/repos?apiVersion=2022-11-28#get-a-repository). Is it something you would consider in the future?

Scalar looks very promising. Love the design. Keep up the great work!

**Comment by marclave** [comment]
Appreciate the kind words, and the request seems good! Lets look into this @cameronrohani @hanspagel ✨

**Comment by hanspagel** [comment]
we got this a while ago already :)

<img width="647" height="269" alt="Screenshot 2026-08-10 at 15 02 55" src="https://github.com/user-attachments/assets/66ad3a00-528c-4837-b40f-d9abd45012e4" />
